website/docs/docs/core/source-install.md

```diff
@@ -12,28 +12,58 @@ Downloading and building dbt Core will enable you to contribute to the project b
 
 ### Installing dbt Core
 
-To install `dbt-core` from the GitHub code source:
+Beginning in v1.8, you must install both dbt Core and the desired adapter because dbt-core has been decoupled from the adapter installation.
```

Comment:
```suggestion
Beginning in v1.8, you must install both dbt Core and the desired adapter because `dbt-core` has been decoupled from the adapter installation.
```